FileHandler.cs

```diff
@@ -0,0 +1,63 @@
+﻿using HwCreateGame;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+
+namespace HwCreateGame
+{
+    public class FileHandler
+    {
+        private string filePath;
+        private Mutex mutex;
+
+        public FileHandler(string filePath)
+        {
+            this.filePath = filePath;
+            mutex = new Mutex();
```

Comment:
What happens if an error happens here? Mutex is forever lost?

Reply:
Your class holds (wrapper for) an unmanaged resource, what does it have to implement when it works with such objects?